src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -1134,3 +1135,24 @@ TEST_CASE("SoftminFunctionTest", "[ActivationFunctionsTest]")
   CheckSoftminDerivativeCorrect(activationData,
                                 desiredDerivatives);
 }
+
+/**
+ * Basic test of the Hard Swish function.
+ */
+TEST_CASE("HardSwishFunctionTest", "[ActivationFunctionsTest]")
+{
+  // Randomly generated data.
+  const arma::colvec activationData("3.6544 -1.9714 -5.2277 1.5448 2.1164");
+
+  // Hand Calculated Values.
+  const arma::colvec desiredActivations("3.6544 -0.3379636 0.0 \
+                                         1.1701345 1.8047248");
+
+  // Hand Calculated Values.
```

Comment:
```suggestion
  // Hand calculated values.
```

To be consistent with the rest of the codebase.